1-first-unix-commands.sh

```diff
@@ -0,0 +1,38 @@
+# q1
+# first approach
+echo "damian" > /home/damianti/firstFile.txt
+# second approach
+touch /home/damianti/secondfile.txt
+echo "damian" >> /home/damianti/secondfile.txt
```

Comment:
You are effectively writing to the file in the same way. What other ways are there to write to a file?